neuro_san/message_processing/basic_message_processor.py

```diff
@@ -88,6 +88,8 @@ def get_compiled_answer(self) -> str:
         structure: Dict[str, Any] = self.get_structure()
         if structure is not None:
             string_struct: str = json.dumps(structure, indent=4, sort_keys=True)
+            if compiled is None:
+                compiled = ""
```

Comment:
If a None string comes over the wire for the message text, start out with an empty string so the subsequent additions to it are fine.